beets/autotag/mb.py

```diff
@@ -202,6 +202,19 @@ def _flatten_artist_credit(credit):
     )
 
 
+def _get_remixer_names(relations):
```

Comment:
Maybe we should very slightly generalize this for future-proofing… we could make it take the relation type as a parameter, and make it default to `'remixer'` for now? I can imagine wanting to collect other relationships in a similar way.